src/pywrap/PyPandasDataFrame.jl

```diff
@@ -102,6 +102,6 @@ function _columns(df, columnnames, columntypes)
     # output a table
     # TODO: realising columns to vectors could be done lazily with a different table type
     schema = Tables.Schema(colnames, coltypes)
-    coldict = Dict(k=>v for (k,v) in zip(colnames, columns))
+    coldict = Tables.OrderedDict(k=>v for (k,v) in zip(colnames, columns))
```

Comment:
Looks good thanks. It unfortunately uses the non-API `Tables.OrderedDict`, but then we're already using the non-API constructor for `Tables.DictColumnTable` so 🤷‍♂️

Reply:
Yeah, it's not great but I figured it would be easier on everyone... it's not really an ideal API on the `Tables` end.

Reply:
I'll think about a PR to Tables to give some of its table types public constructors.